src/org/parosproxy/paros/core/proxy/ProxyThread.java

```diff
@@ -270,7 +271,10 @@ public void run() {
 	    	if (firstHeader != null) {
 	    		log.warn("Timeout accessing " + firstHeader.getURI());
 	    	} else {
-	    		log.warn("Timeout", e);
+	    		log.warn("Socket Timeout");

```

Comment:
Worth changing to "Timed out while reading first message.", this is a special case.